### ISSUE TITLE
Devil

### ISSUE THREAD

**Issue by Fenyx4** [cambragol/advent-of-the-trinity#113]
Whaaaaat!?!

**Comment by cambragol** [comment]
Is that a "Whoaa..that's crazy...", or a "This is an outrage!" kind of a Whaaaat?

**Comment by Fenyx4** [comment]
It was a bit of "what just happened" and a lot of "what?! this is awesome!" with a smattering of "how did you do that?"

**Comment by cambragol** [comment]
I wanna say more but don't want to spoil it for the others. 

I assume you didn't survive the encounter?

**Comment by Fenyx4** [comment]
Oh I should give more information on this now that we have spoiler tags;

<details> 
  <summary>Devil - Spoiler</summary>
  I ran into them only once my entire time in the Underdeep. I was lucky and ran into them when I was in good shape (characters were topped up on mana and health) so I did survive.
  I think it also helped that I started off the first round by casting Negate because they looked enough like Balrons that I feared a sleep salvo.
  Then the negate wore off and they began hitting me with Tremor! Ow!
  But I had killed off enough of them during the negate that I survived.
  
  If I had fought them at other times in the Underdeep I would have been in a lot more trouble. There were many times when I was being harrowed by groups of monsters and was having problems keeping my mana high enough to keep everyone healed. If I had run into the Devils then I would not have fared as well. 
  
  I was sad I never ran into them again. Was hoping to find out more of their abilities.
</details>

<details> 
  <summary>Spoiler - General Underdeep exploration</summary>
  I managed to map out (I think) the entire Underdeep. If I remember correctly I went through ~250 heal spells to do so.
  I found two Moon Orbs. A third spot that looked like it should have one didn't have anything.
</details>

**Comment by cambragol** [comment]
Wow, great feedback! I'll use the spoiler tag too for now:

<details> 
  <summary>Devil Spoiler</summary>
Sounds like you had a perfect encounter with them! But, you may have missed some of their other abilities. That is a real bummer you only encountered them on time. I will have to check that out, as a few more encoutners would have made things better, obviously. There is a lot of code in those guys, and only meeting them once, ever, knocks some of the point out of that code.

I like the sound of your adventures in the Underdeep. Certainly not meant to be a easy time down there.
</details>

<details> 
 <summary>Underdeep Spoiler</summary>
You found everything. The third spot is empty, as that is where Lord British's moonstone was found. There is a quest dialogue explaining that, somewhere. I guess you missed that. Nevertheless, you have 3 Orbs now, right?

Also, there is more to the underdeep, but it is inacessible in a regular game, and is there for completeness. areas around the Abyss etc. are there. Maybe overlay your map onto the map of Britannia and you can see the logic in it all.
</details>

**Comment by Fenyx4** [comment]
<details> 
  <summary>Spoiler - Underdeep & Other stuff about my progress overall</summary>
  Nope! Still only two orbs.
  I don't have any artifacts either. And I haven't found the book yet.
  I only have two of the dungeon sealing words.
  I have all but Ash for reagent words (Although I'm pretty sure none of them work.)
  But reagent words led me to Merlock who gave me something else to yell. But I haven't figured out what it does.
  I did get the ring though.
  I'm doing another sweep on all the towns now. As I missed some stuff my first time through!
  
  I had already done the overlay. :) The design of the Underdeep is very cool!
</details>

**Comment by cambragol** [comment]
More great feedback!

Might have let a clue slip with the orbs...oops.

Looks like you have a truck load of content to go through yet! The magic system has been cracked by another beta tester, but maybe you'll be the second. It has some small bugs, but it is crackable.

On the ring, there is another thread discussing it and its function: #148 -- maybe chime in there.

The Underdeep and magic system are being discussed here: #136 -- already have a new location there, which will be in the next Beta.
Glad you like the design. Some time and thought went into it. Works out I think.

**Comment by MrKnitster** [comment]
Ran into my first group of devils. Yikes! Tremors just mow through the party.

**Comment by cambragol** [comment]
Yay! Hopefully they are making travel in the Underdeeps deadly.

The devils once had a larger part in the story of the trinity, but I removed a bunch of stuff to make the mod finishable, and not as grand as the main quest (avatar/abyss). Had to keep the devils though, as they are just too cool, and a new enemy is fun.

**Comment by Fenyx4** [comment]
I was just sailing past Avatar Isle and a Devil came flying off the island and flew after me. o_0

**Comment by cambragol** [comment]
Yes..I have them spawning in lava, so they could spawn there. I am not sure about that...a hint of what lies below? My gut feeling is to restrict them to the underdeep only. 


what do you think?

**Comment by cambragol** [comment]
Have either of you seen Devils using 'Summon' to summon Balrons and Daemons? I just noticed it is not working in current Beta and I am not sure why, or how long, this was broken.

**Comment by cambragol** [comment]
Well, I got it working again, but I have a feeling you guys may not have felt the full strength of the Devils, with mutiple tremor, summon and sleep spells coming at you every round.

The problem was another quick with random numbers, which I cannot explain. u4_rnd1(3) doesn't work, but u4_rnd1(2) does. Very odd. But now it works.

**Comment by Fenyx4** [comment]
I have not seen them summon.

But I have only fought two groups of them.

**Comment by cambragol** [comment]
Yeah, it looks like I 'fixed' the way they were choosing to Summon or Tremor, and in doing so took away their summon ability. Now with Summon and Sleep working they are a lot more deadly.

I also just spent 2 hours typing up a list of updates for Beta 12. Then my laptop ran out of batteries. slightly miffed.

**Comment by MrKnitster** [comment]
I definitely saw them summoning (only daemons, I think). Lots of tremors, too. I don't think I've seen them cast sleep.
